client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -94,10 +96,27 @@ export class ProductPurchaseFeaturesList extends Component {
 		const {
 			isPlaceholder,
 			plan,
+			currentPlan,
 			planHasDomainCredit,
 			selectedSite,
 			showCustomizerFeature,
+			scheduleId,
 		} = this.props;
+
+		let hasBusinessOnboardingExpired;
+		if ( currentPlan ) {
+			const expiryDateMoment = this.props.moment( currentPlan.expiryDate );
+			const businessOnboardingExpiration = this.props.moment( PLAN_BUSINESS_ONBOARDING_EXPIRE );
+
+			hasBusinessOnboardingExpired = businessOnboardingExpiration.diff( expiryDateMoment ) < 0;
+		}
+
+		const hasIncludedSessions = scheduleId === 1;
+		const hasPurchsedSessions = scheduleId > 1;
```

Comment:
We can make this more specific. Presently scheduleId === 2 is for purchased sessions.

Reply:
Found a procedure to test the QS functionality without bothering the HE ("End-to-end test without bothering HEs"). Following it would result in the scheduleId having a specific, positive, value which should be covered with > 1.